models/event.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var EventSchema = new mongoose.Schema({
```

Comment:
Unless this changed since you added this comment, this is the OSDI event schema.